webapp/tests/model/test_form_data.py

```diff
@@ -220,13 +222,18 @@ def setUp(self) -> None:
             'request_new_tax_number': 'yes'
         }
 
-        self.married_familienstand = {
+        self.familienstand_without_account_holder = {
```

Comment:
Naming problem then. Maybe `married_familienstand_without_account_holder` would be better.

This attribute is a proper married_familienstand. However, if we have married users, we need to set the account_holder. Otherwise, we get an error raised.
To keep the old test behavior, I added this new attribute without the account holder

Reply:
yes, but why not have self.married_familienstand and self.valid_iban_married and self.valid_iban_single. We're putting that together anyways? I was just confised, because for me, `familienstand` ist a very diefferent thing (because different page) than the iban data.